jenkins/scripts/slurm_run.sh

```
@@ -85,7 +84,24 @@ export LD_LIBRARY_PATH=$containerLDLibPath
 echo "Library Path:"
 echo "$LD_LIBRARY_PATH"
 env | sort
-fullCmd="${testCmdLines[*]}"
 echo "Running: $testCase"
-echo "Full Command: $fullCmd"
-eval $fullCmd
+echo "Full Command: $pytestCommand"
+eval $pytestCommand
+
+if [ "$perfMode" = "true" ]; then
+    if [[ "$" == *PyTorch* ]]; then
```

Shellcheck (0.10.0)

[warning] The arguments to this comparison can never be equal. Make sure your syntax is correct.

(SC2193)